test/circuit/library/initial_states/test_fermionic_gaussian_states.py

```diff
@@ -0,0 +1,116 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test fermionic Gaussian state preparation circuits."""
+
+from test import QiskitNatureTestCase
+
+import numpy as np
+from qiskit.quantum_info import random_hermitian, Statevector
+from qiskit_nature.circuit.library import FermionicGaussianState, SlaterDeterminant
+from qiskit_nature.converters.second_quantization import QubitConverter
+from qiskit_nature.mappers.second_quantization import JordanWignerMapper
+from qiskit_nature.operators.second_quantization.quadratic_hamiltonian import QuadraticHamiltonian
+
+
+def _random_antisymmetric(dim: int):
+    mat = np.random.randn(dim, dim) + 1j * np.random.randn(dim, dim)
+    return mat - mat.T
+
+
+class TestSlaterDeterminant(QiskitNatureTestCase):
```

Comment:
This file is called `test_fermionic_gaussian_states` yet it appears to focus on testing the `SlaterDeterminant` class (as the name suggests). Please split the tests out into two properly named files with distinct sets of tests as this helps with the code+test navigation.

Reply:
Actually both classes were tested together to reduce code duplication. But I've split them into separate files now.